sci-electronics/librepcb/librepcb-0.1.0.ebuild

```diff
@@ -1,41 +1,34 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit qmake-utils
 
 DESCRIPTION="Free EDA software to develop printed circuit boards"
 HOMEPAGE="https://librepcb.org/"
-MY_P="LibrePCB-first_pcb"
-SRC_URI="https://github.com/LibrePCB/LibrePCB/archive/first_pcb.tar.gz -> ${MY_P}.tar.gz"
+#MY_P="LibrePCB-${PV}"
```

Comment:
if the commented code is unnecessary, please remove it.

Reply:
done